en/docs/get-started/quick-start-guide.md

```diff
@@ -1,124 +1,134 @@
-- Consuming the published REST API via an external application.
-- Releasing a new version of the REST API with added functionality.
 
-Let's consider a use case where a web application developer designs an application that allows you to maintain reading lists. A user can create a private reading list by adding books. The user can also view and delete them when required. A Choreo developer develops a REST API for this web application to consume. The two developers will create two versions of the solution in two iterations. The second version of the application will add data persistence as a new feature.
+This guide walks you through the following steps:
 
-The following diagram summarizes the use case:
-
-![Use case summary](../assets/img/get-started/use-case-summary.png){.cInlineImage-full}
-
-Let's get started!
+- Develop, deploy, and test a service component.
+- Publish the service endpoint as a REST API for your web application to consume. 
+- Expose the REST API via Choreo API management.
+- Securely consume the published REST API via your web application.
+- Deploy your web application and use Asgardeo as the IDaaS provider to secure user authentication to the web application.
+ 
 
 ## Prerequisites
 
 Before you try out this guide, complete the following:
 
-1. Create a GitHub repository to save the REST API implementation. For this guide, you can fork [https://github.com/wso2/choreo-examples/tree/version-1](https://github.com/wso2/choreo-examples/tree/version-1).
+1. Create a GitHub repository to save the service implementation. For this guide, you can fork [https://github.com/wso2/choreo-examples](https://github.com/wso2/choreo-examples).
 
-2. If you are logging in to Choreo Console for the first time, create an organization as follows:
+2. If you are signing in to the Choreo Console for the first time, create an organization as follows:
 
-    1. In [https://wso2.com/choreo/](https://wso2.com/choreo/), click **Try Choreo Now**.
+    1. Go to [https://wso2.com/choreo/](https://wso2.com/choreo/) and click **Try Choreo Now**.
 
     2. Sign in to Choreo using your Google, GitHub, or Microsoft account.
 
     3. Enter a unique organization name. For example, `Stark Industries`.
 
         ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
 
-    4. Read and accept the privacy policy and terms of use before you proceed.
+    4. Read and accept the privacy policy and terms of use.
 
     5. Click **Create** to add the new organization.
 
-    You will be viewing the **Home** page of the Choreo Console.
+    This takes you to the **Home** page of the Choreo Console.
+
+    !!! info "Enable Asgardeo as the key manager"
+         
+         If you created your organization in Choreo before the 21st of February 2023, and you have not already enabled Asgardeo as the key manager, follow these steps to enable Asgardeo as the default key manager:
```

Comment:
remembering dates is difficult, isn't there anyother way for them to check if they have enabled Asgardeo as the default Key Manager?

Reply:
AFAIK there is a banner if a user has not enabled Asgardeo as KM. This note is applicable to the very few who have not enabled it even with the banner.

Reply:
@Asitha is there any other way that we can mention to help users to check whether they have enabled Asgardeo as KM?